src/main/resources/view/MainWindow.fxml

```diff
@@ -45,7 +45,7 @@
                     </padding>
                 </StackPane>
 
-                <VBox fx:id="trackedModuleList" styleClass="pane-with-border" minWidth="340" prefWidth="340"
+                <VBox fx:id="ModuleList" styleClass="pane-with-border" minWidth="340" prefWidth="340"
```

Comment:
Try to minimize unnecessary modifications to the master.

Reply:
Corrected.